### ISSUE TITLE
Some ids are not unique for annotations

### ISSUE THREAD

**Issue by adrianboguszewski** [pedropro/TACO#12]
Hi,

I discovered that some are not unique for annotations e.g. 309 and 4040 occur two times. 
I consider... is there a race condition during parallel labeling?

**Comment by pedropro** [comment]
You're right. Thanks for bringing this up. I am going to investigate this issue and correct the annotation file.

**Comment by CodeSammich** [comment]
This is still a bug. Annotation id 309 still has a duplicate

**Comment by ERICMIAO0817** [comment]
Same here，wonder if there is a solution(besides fixing id one by one)?
Appreciate it!!!

**Comment by elmaestrobert** [comment]
Also encountering this issue now

**Comment by ken011528** [comment]
Thank You , there maybe two id are duplicated. "id": 309 and "id": 4040.
